tests/integration/modules/mac_service.py

```diff
@@ -70,17 +63,18 @@ def test_launchctl(self):
         '''
         # Expected Functionality
         self.assertTrue(
-            self.run_function('service.launchctl', ['error', 'bootstrap', 64]))
+            self.run_function('service.launchctl',
+                              ['error', 'bootstrap', '64']))
         self.assertEqual(
             self.run_function('service.launchctl',
-                              ['error', 'bootstrap', 64],
+                              ['error', 'bootstrap', '64'],
```

Comment:
This is a legit bug in the `cmdmod` module. I have corrected it in #36254. Therefore, we can go ahead and merge this since the bug it has exposed is now resolved.